apps/web-portal/tests/preview.test.mjs

```diff
@@ -0,0 +1,36 @@
+import sinon from 'sinon'
+import test from '@zorko-io/tool-test-harness'
+import {createClient, ClientTypes} from '@zorko-io/util-web-api-client'
+import {MockLogger} from '@zorko-io/util-logger'
+import {WebPortalExpressApp} from '../lib'
+import {config} from '../config'
+
+test.beforeEach((t) => {
```

Comment:
I see a duplication of tests setup, let's try to extract before and after in helper function like 'setupAppContext'